redfish-core/lib/assembly.hpp

```diff
@@ -534,6 +534,26 @@ inline void setAssemblylocationIndicators(
                                 return;
                             }
                             messages::success(asyncResp->res);
+
+                            // Once the CM has been done, reset the fault LED.
+                            crow::connections::systemBus->async_method_call(
+                                [asyncResp](
+                                    const boost::system::error_code ec2) {
+                                    if (ec2)
+                                    {
+                                        BMCWEB_LOG_ERROR
+                                            << "Failed to reset fault LED"
+                                            << ec2;
+                                        messages::internalError(asyncResp->res);
+                                        return;
+                                    }
+                                },
+                                "xyz.openbmc_project.LED.GroupManager",
```

Comment:
We should use D-Bus associations to get the group object. I think you can use something similar to setLocationIndicatorActive in led.hpp and get it to look for the fault association instead.

Reply:
Okay.